stl/inc/xutility

```diff
@@ -119,9 +119,21 @@ struct _Get_rebind_alias<_Ty, _Other, void_t<typename _Ty::template rebind<_Othe
     using type = typename _Ty::template rebind<_Other>;
 };
 
+// FUNCTION TEMPLATE construct_at

```

Comment:
In an earlier version I used `construct_at` for `_Construct_in_place` ThatswhyI moved it. Is this really necessary?

Reply:
If the compilers are going to allow True Placement New - which I believe they are - then we should probably just use `_Construct_in_place` for code that needs to work in pre-C++20 non-`constexpr` modes.